app/constants/constants.js

```diff
@@ -178,7 +180,21 @@ export const CONNECTION_CONFIG = {
             'value': 'secretAccessKey',
             'type': 'password'
         }
-    ] // TODO - password options for apache drill
+    ], // TODO - password options for apache drill
+    [DIALECTS.DATA_WORLD]: [
+        {
+            'label': 'Dataset/Project URL',
```

Comment:
I'd like to change this. Instead of having the user to input an URL and the connector to parse the URL to determine `owner` and dataset `id`. I'd like the user to input `owner` and `id`:
- this is more consistent with the existing connectors
- and it simplifies the code (no need for `getPathNames`)

Reply:
Hi, @n-riesco, unless you feel strongly about this, we would like to use the URL. That would be consistent with our other connectors work (e.g. Tableau and Google Data Studio) and is more usable, as users only have one value to copy and paste, which is readily available on the browser for the dataset they are looking at. We do a little bit of extra work in code to spare the user in real life. Also, using URIs that contain information to parse out isn't without a precedent, the most common is JDBC.

Reply:
No, I don't feel strongly about it.

@tarzzz @chriddyp are you OK with using an URL here?

Reply:
>  is more usable, as users only have one value to copy and paste, which is readily available on the browser for the dataset they are looking at

This sounds like a good reason to me 👍